### DISCUSSION TITLE
How to connect to postgresql from Nue?

### DISCUSSION THREAD

**Discussion by avitkauskas** [nuejs/nue#238] [Q&A]
I'd like to build an app that reads data from the postgresql database.
I'd like to use this postgres client: https://github.com/porsager/postgres
Installed it with `bun install postgres` and created a file `db.ts` in the root of the project with `import postgres from 'postgres'` in it. I then import it in my models file to use sql there. But I get an error:
```
TypeError: Failed to resolve module specifier "postgres". Relative references must start with either "/", "./", or "../".
```
What I do wrong?

**Comment by avitkauskas** [comment]
Ahrr... All Nue code is frontend and goes into the browser. So I need something separate for the backend. Correct?

**Reply by nobkd** [reply]
Might be related, as it is about server side components:

- #231
- #234
